seed/static/seed/partials/analyses.html

```diff
@@ -33,6 +33,7 @@ <h2><i class="fa fa-bar-chart"></i> {$:: 'Analyses' | translate $}</h2>
                                 <i class="glyphicon glyphicon-trash" aria-hidden="true"></i> -->
                                 <i class="glyphicon glyphicon-play" title="Start Analysis" aria-hidden="true" ng-if="analysis.status === 'Ready'" ng-click="start_analysis(analysis.id)"></i>
                                 <i class="glyphicon glyphicon-stop" title="Stop Analysis" aria-hidden="true" ng-if="['Pending Creation', 'Creating', 'Queued', 'Running'].indexOf(analysis.status) >= 0" ng-disabled="['Pending Creation', 'Queued'].indexOf(analysis.status) >= 0" ng-click="stop_analysis(analysis.id)"></i>
+                                <i class="glyphicon glyphicon-trash" title="Delete Analysis" aria-hidden="true" ng-click="delete_analysis(analysis.id)"></i>
```

Comment:
Not sure when we want to hide/disable this icon. I just left it showing and enabled for all analysis statuses.

Reply:
I think it's fine to display at all times like you have it now.